plugins/ledmatrix-weather/test_almanac_moon_data.py

```
@@ -0,0 +1,134 @@
+def check_moonset_fallback():
+    """astral fails on 2026-06-11; the fallback recovers the real afternoon
+    moonset (independently confirmed ~17:23 local)."""
+    plugin = object.__new__(WeatherPlugin)
+    fails = []
+    d = datetime.date(2026, 6, 11)
+
+    # Document the underlying astral bug: it raises on this date.
+    astral_raised = False
+    try:
+        astral_moon.moonset(OBS, d, tzinfo=TZ)
+    except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)